### PR TITLE
fix(ChatMessage): prevent hover over invisible actions area

### PR DESCRIPTION
**Pull request by kuzhelov**
## TODO
- [x] update changelog

---------

This PR fixes undesired effect when actions menu is showed up while user hovering over invisible to her area. This has resulted in weird user experience (consider on-hover appearance of actions menu for the lower chat message):

chat-message-problem

### CODE REVIEW

**Comment by jurokapsiar** [comment]
focus and hover styles (when you hover on the right edge of the message) are broken after the change:

image

**Comment by kuzhelov** [comment]
@jurokapsiar, thanks a lot for spotting that - have updated styles to address these issues:

chat-message-problem

## submenu's accessibility

chat-message-problem

**Review by jurokapsiar — APPROVED** [review]

File: packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts [hunk 1]
```diff
@@ -58,10 +59,14 @@ const chatMessageStyles: ComponentSlotStylesInput<
...
     boxShadow: v.actionMenuBoxShadow,
-    opacity: p.focused ? 1 : 0,
     position: 'absolute',
     right: v.actionMenuPositionRight,
     top: v.actionMenuPositionTop,
+    overflow: p.focused ? 'visible' : 'hidden',
+
+    // squash actions menu to prevent accidental hovers over its invisible area
+    width: p.focused ? 'auto' : 0,
+    opacity: p.focused ? 1 : 0,
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts:76]: nit: reorder `opacity` and `width` by alpha order 🐱

**Review by layershifter — APPROVED** [review]
Don't forget about a changelog entry 🐱

**Pull request merged**